### PR TITLE
[tpm2.0] Add tpm2 support in installer

### PR DESCRIPTION
**Pull request by crogers1**
OXT-938

Signed-off-by: Chris Rogers <rogersc@ainfosec.com>

### CODE REVIEW

File: part2/stages/TPM-Check [hunk 1]
```diff
@@ -50,7 +50,14 @@ update_gauge() {
 }
 # dump attributes of the systems TPM into the log
 log_tpm_attrs() {
-    tpm_version >&2
+    is_tpm_2_0
+    tpm2=$?
+    if [ "${tpm2}" -eq 0 ];
+    then
+	    tpm2_dump_capability --capability=properties-fixed >&2
```
Comment (jean-edouard) [inline part2/stages/TPM-Check]: Tab spotted! :)

**Comment by rossphilipson** [comment]
@crogers1  this one needs a stable-7 cherry pick

**Comment by crogers1** [comment]
Done and done.

**Comment by rossphilipson** [comment]
Merging soon

**Pull request merged**